rollup.config.js

```diff
@@ -7,32 +19,22 @@ export default [
     file: './dist/godirect.min.js',
     format: 'esm'
   },
-  plugins: [terser({
-    output: function(node, comment) {
-      const { type } = comment;
-      if (type === 'comment2') {
-        // multiline comment
-        return /@preserve|@license/;
-      }
-      return undefined;
-    }
-  })]
+  plugins: [terserPlugin]
 },
 {
   input: './src/godirect.js',
   output: {
-    file: './dist/godirect.min.cjs.js',
-      format: 'cjs'
+    file: './dist/godirect.min.umd.js',
+    format: 'umd',
+    name: 'godirect'
   },
-  plugins: [terser({
-    output: function(node, comment) {
-      const { type } = comment;
-      if (type === 'comment2') {
-        // multiline comment
-        return /@preserve|@license/;
-      }
-      return undefined;
-    }
-  })]
+  plugins: [
+    terserPlugin,
+    babel({
+      babelrc: false,
+      presets: ['@babel/env'],
+      exclude: 'node_modules/**'
+    })
```

Comment:
Hi @jordanaustin, I had to really read into babel/rollup config options to build a bundle with the right transformations for node in our case. It turns out this is what we need:
```
    babel({
      presets: [
        ["@babel/env", {
	  "targets": {
	    "node": "8.0.0"
	  },
          "modules": false,
          "useBuiltIns": "usage"
        }]
      ],
      exclude: 'node_modules/**',
    })

```

This also requires a new dependency in package.json -- specifically a dependency not dev dependency, we need it to be installed when requiring @vernier/godirect:
```
  "dependencies": {
    "core-js": "^2.6.1"
  }
```

Reply:
That's annoying, Node 8 has support for async/await - I wonder why Scratch3 doesn't allow async/await